src/components/Section/Identification/ApplicantBirthDate/ApplicantBirthDate.jsx

```diff
@@ -156,13 +170,13 @@ export class ApplicantBirthDate extends Subsection {
     )
   }
 }
-
+const TODAY = DateTime.local()
```

Comment:
Can we move this inside of the component so that it's not in the global scope? This way, it'll be scoped within the component and it'll be defined because the variable is used.

Reply:
done